OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/OSOutcomeEventsV1Service.java

```diff
@@ -0,0 +1,20 @@
+package com.onesignal.outcomes;
+
+import com.onesignal.OneSignalAPIClient;
+import com.onesignal.OneSignalApiResponseHandler;
+
+import org.json.JSONObject;
+
+class OSOutcomeEventsV1Service extends OSOutcomeEventsClient {
+
+    OSOutcomeEventsV1Service(OneSignalAPIClient client) {
+        super(client);
+    }
+
+    /***
+     * API endpoint /api/v1/outcomes/outcomes/measure
```

Comment:
"outcomes" is repeated in the comment.
`/api/v1/outcomes/outcomes/measure` ->  `/api/v1/outcomes/measure`

Reply:
good catch!